src/agenteval/leaderboard/models.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel, Field
+
+from ..models import SuiteConfig, TaskResult, SubmissionMetadata
+
+class LeaderboardSubmission(BaseModel):
```

Comment:
This is the old `EvalResult` class, now defined explicitly as the schema expected by the leaderboard

Reply:
This class structure could be simplified, since `TaskResult` has more information than the leaderboard needs.

Ideally, all of the transforms done in `leaderboard.view._get_dataframe()` would be handled by `lb publish`, leaving this as a pure data class with no processing necessary by the view code and a simpler `README` schema